common/lib/protocol-definitions/api-report/protocol-definitions.api.md

```diff
@@ -195,6 +195,11 @@ export interface IHelpMessage {
     version?: string;
 }
 
+// @public (undocumented)
+export interface ILocalSequencedClient extends ISequencedClient {
```

Comment:
is this needed by the server? only contracts between client and server should be in protocol defs

Reply:
That's a good point, actually. I think I'd better put it in `protocol-base`, as that's where there would actually be a need for it outside of `container-loader`, which already has a dep on `protocol-base`